sklearn/tests/test_cross_validation.py

```diff
@@ -1106,6 +1106,16 @@ def bad_cv():
     assert_raises(ValueError, cval.cross_val_predict, est, X, y, cv=bad_cv())
 
 
+def test_sparse_fit_params():
+    boston = load_boston()
+    X, y = boston.data, boston.target
+    train, test = next(iter(cval.KFold(len(boston.target))))
+    clf = MockClassifier()
+    fit_params = {'sparse_sample_weight': coo_matrix(np.eye(X.shape[0]))}
+    a = cval._fit_and_predict(clf, X, y, train, test, 0, fit_params)
```

Comment:
I think it would be more meaningful to write a non-regression test that works with the public API (e.g. `cval.cross_val_score`) instead of directly tapping into the private API such as `cval._fit_and_predict`.